packages/node/test/manual/release-health/runner.js

```diff
@@ -15,7 +15,7 @@ for (const dir of scenariosDirs) {
 
 const processes = scenarios.map(([filename, filepath]) => {
   return new Promise(resolve => {
-    const scenarioProcess = spawn('node', [filepath]);
+    const scenarioProcess = spawn('node', [filepath], { timeout: 10000 });
```

Comment:
Noticed this was never timing out, so adding a reasonable timeout here to prevent super long test run times when something goes wrong.